vendor/bat-native-ads/data/resources/catalog-schema.json

```diff
@@ -61,6 +61,9 @@
           "priority": {
             "type": "number"
           },
+          "ptr": {
```

Comment:
Catalog version for `ptr` will be 4 (search for `"/v3/catalog"` to see tests / static_values.h which need updating